<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>500</height>
   </size>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Figures</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="createFigureWidget" native="true">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>0</y>
      <width>151</width>
      <height>301</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background: rgb(255, 243, 230)</string>
    </property>
    <widget class="QPushButton" name="triangleCreatePushButton">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Треугольник</string>
     </property>
    </widget>
    <widget class="QPushButton" name="circleCreatePushButton">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>70</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Окружность</string>
     </property>
    </widget>
    <widget class="QWidget" name="triangleCreateWidget" native="true">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>120</width>
       <height>176</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>120</width>
       <height>16777215</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Сторона A:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="triangleSideA">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Сторона B:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="triangleSideB">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Сторона C:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="triangleSideC">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="creatingTriangleButton">
        <property name="text">
         <string>Создать</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="circleCreateWidget" native="true">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>120</width>
       <height>86</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>120</width>
       <height>16777215</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item alignment="Qt::AlignLeft">
       <widget class="QLabel" name="label_17">
        <property name="text">
         <string>Радиус:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="circleRadius">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="creatingCircleButton">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Создать</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="labelDataError">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>280</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Ошибка!</string>
     </property>
    </widget>
    <zorder>triangleCreatePushButton</zorder>
    <zorder>circleCreatePushButton</zorder>
    <zorder>circleCreateWidget</zorder>
    <zorder>triangleCreateWidget</zorder>
    <zorder>labelDataError</zorder>
   </widget>
   <widget class="QWidget" name="bottomWidget" native="true">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>460</y>
      <width>521</width>
      <height>51</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background: rgb(255, 243, 230)</string>
    </property>
    <widget class="QLineEdit" name="triangleSideALabel">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="triangleSideBLabel">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="triangleSideCLabel">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>20</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>20</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgb(209, 209, 209)</string>
     </property>
     <property name="text">
      <string>След.</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="circleRadiusLabel">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="prismHeightLabel">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>20</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>0</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Текущая фигура:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonPrevFig">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>20</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgb(209, 209, 209)</string>
     </property>
     <property name="text">
      <string>Пред.</string>
     </property>
    </widget>
    <widget class="QPushButton" name="makePrismButton">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>0</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgb(209, 209, 209)</string>
     </property>
     <property name="text">
      <string>Сделать призмой</string>
     </property>
    </widget>
    <widget class="QPushButton" name="scalePlusButton">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>0</y>
       <width>20</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgb(209, 209, 209)</string>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
    <widget class="QPushButton" name="scaleMinusButton">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>20</y>
       <width>20</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgb(209, 209, 209)</string>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonDelFigure">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>20</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgb(209, 209, 209)</string>
     </property>
     <property name="text">
      <string>Удалить</string>
     </property>
    </widget>
    <widget class="QPushButton" name="figuresPrismsSwitcher">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>0</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgb(209, 209, 209)</string>
     </property>
     <property name="text">
      <string>Призмы</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>-1</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>нет</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="createFigureButton">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>10</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Создать фигуру</string>
    </property>
   </widget>
   <widget class="QWidget" name="prismCreateWidget" native="true">
    <property name="geometry">
     <rect>
      <x>339</x>
      <y>399</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: rgb(255, 243, 230)</string>
    </property>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Введите высоту:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="prismHeightInput">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>10</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="createPrism">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>35</y>
       <width>80</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Создать</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelPrismDataError">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Ошибка!</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widgetFigureCalcs" native="true">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>320</y>
      <width>151</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: rgb(255, 243, 230)</string>
    </property>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Площадь</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="figureSpace">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>10</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="figurePerimeter">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>40</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Периметр</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="prismCalcs" native="true">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>300</y>
      <width>151</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: rgb(255, 243, 230)</string>
    </property>
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>1</x>
       <y>-1</y>
       <width>71</width>
       <height>101</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>S. Осн.</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Объем</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>S. Пов.</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QFrame" name="frame_2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>0</y>
       <width>101</width>
       <height>101</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLineEdit" name="labelPrismBaseSpace">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="labelPrismVolume">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="labelPrismSpace">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QPushButton" name="moveUp">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>400</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>^</string>
    </property>
   </widget>
   <widget class="QPushButton" name="moveDown">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>440</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>v</string>
    </property>
   </widget>
   <widget class="QPushButton" name="moveLeft">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>420</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="moveRight">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>420</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;</string>
    </property>
   </widget>
   <widget class="GLPrismsWidget" name="prismsWidget" native="true">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>501</width>
      <height>501</height>
     </rect>
    </property>
    <widget class="QPushButton" name="rotateLeft">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>420</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="rotateRight">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>420</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="rotateUp">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>400</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>^</string>
     </property>
    </widget>
    <widget class="QPushButton" name="rotateDown">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>440</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>v</string>
     </property>
    </widget>
    <widget class="QLabel" name="rotateLabel">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>370</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgb(255, 243, 230)</string>
     </property>
     <property name="text">
      <string>Поворот</string>
     </property>
    </widget>
    <widget class="QLabel" name="moveLabel">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>370</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgb(255, 243, 230)</string>
     </property>
     <property name="text">
      <string>Двигать</string>
     </property>
    </widget>
    <widget class="QPushButton" name="heightMinus">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>400</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>h-</string>
     </property>
    </widget>
    <widget class="QPushButton" name="heightPlus">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>400</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>h+</string>
     </property>
    </widget>
   </widget>
   <widget class="GLFiguresWidget" name="figuresGlWidget" native="true">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>501</width>
      <height>501</height>
     </rect>
    </property>
   </widget>
   <zorder>figuresGlWidget</zorder>
   <zorder>prismsWidget</zorder>
   <zorder>createFigureWidget</zorder>
   <zorder>bottomWidget</zorder>
   <zorder>createFigureButton</zorder>
   <zorder>prismCreateWidget</zorder>
   <zorder>widgetFigureCalcs</zorder>
   <zorder>prismCalcs</zorder>
   <zorder>moveUp</zorder>
   <zorder>moveDown</zorder>
   <zorder>moveLeft</zorder>
   <zorder>moveRight</zorder>
  </widget>
  <action name="actionmoveUp">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>moveUp</string>
   </property>
  </action>
  <action name="actionmoveDown">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>moveDown</string>
   </property>
  </action>
  <action name="actionmoveLeft">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>moveLeft</string>
   </property>
  </action>
  <action name="actionmoveRight">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>moveRight</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GLFiguresWidget</class>
   <extends>QWidget</extends>
   <header location="global">glfigureswidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GLPrismsWidget</class>
   <extends>QWidget</extends>
   <header location="global">glprismswidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
